xml/templates/message_templates/event_offline_receipt_html.tpl

```diff
@@ -165,267 +166,250 @@
      {if {event.is_monetary|boolean}}
 
       <tr>
-       <th {$headerStyle}>
-        {event.fee_label}
-       </th>
+        <th {$headerStyle}>
+            {event.fee_label}
+        </th>
       </tr>
-
-      {if !empty($lineItem)}
-       {foreach from=$lineItem item=value key=priceset}
-        {if $value neq 'skip'}
-          {if $lineItem|@count GT 1} {* Header for multi participant registration cases. *}
-           <tr>
-            <td colspan="2" {$labelStyle}>
-             {ts 1=$priceset+1}Participant %1{/ts}
+        {if $isShowLineItems}
+          {foreach from=$participants key=index item=currentParticipant}
+            {if $isPrimary || {participant.id} === $currentParticipant.id}
+              {if $isPrimary && $lineItems|@count GT 1} {* Header for multi participant registration cases. *}
+                <tr>
+                  <td colspan="2" {$labelStyle}>
+                      {ts 1=$currentParticipant.index}Participant %1{/ts} {$currentParticipant.contact.display_name}
+                  </td>
+                </tr>
+              {/if}
+              <tr>
+                <td colspan="2" {$valueStyle}>
+                  <table>
+                    <tr>
+                      <th>{ts}Item{/ts}</th>
+                      <th>{ts}Qty{/ts}</th>
+                      <th>{ts}Each{/ts}</th>
+                        {if $isShowTax && {contribution.tax_amount|boolean}}
+                          <th>{ts}Subtotal{/ts}</th>
+                          <th>{ts}Tax Rate{/ts}</th>
+                          <th>{ts}Tax Amount{/ts}</th>
+                        {/if}
+                      <th>{ts}Total{/ts}</th>
+                        {if !empty($pricesetFieldsCount)}
+                          <th>{ts}Total Participants{/ts}</th>
+                        {/if}
+                    </tr>
+                    {foreach from=$currentParticipant.line_items item=line}
+                      <tr>
+                        <td {$tdfirstStyle}>{$line.title}</td>
+                        <td {$tdStyle} align="middle">{$line.qty}</td>
+                        <td {$tdStyle}>{$line.unit_price|crmMoney:$currency}</td>
+                          {if $line.tax_rate || $line.tax_amount != ""}
+                            <td>{$line.tax_rate|string_format:"%.2f"}%</td>
+                            <td>{$line.tax_amount|crmMoney:$currency}</td>
+                          {else}
+                            <td></td>
+                            <td></td>
+                          {/if}
+                        <td {$tdStyle}>
+                            {$line.line_total+$line.tax_amount|crmMoney:$currency}
+                        </td>
+                        {if !empty($pricesetFieldsCount)}
+                          <td {$tdStyle}>{$line.participant_count}</td>
+                        {/if}
+                      </tr>
+                    {/foreach}
+                    {if $isShowTax}
```

Comment:
This is always redundant in the offline template, because we only ever have one participant and the taxes are detailed immediately below this.

Reply:
@larssandergreen so I did manage to create a pending -> complete scenario where you can have multiple participants.

Also - I tried to use exactly the same code in this as the online cos that seemed like it was more consistent

Reply:
Actually, I guess there's a lot in here that's pretty redundant in the offline context - but maybe it's easier to leave it in so they are more mergeable in the future?

Reply:
Fair enough. I'll do a follow up to simplify this in the most common case where we only have one participant.

Reply:
@larssandergreen so I guess my thinking is that ideally we really don't have different templates - so I was trying to make them the same in any place where there isn't a specific reason for them to differ. Of course I have let the custom field vs profile difference stay - but I had thought about making the case to just show the profiles on back office too, once that code is generalised